pkgs/tools/package-management/nix/default.nix

```diff
@@ -36,16 +38,20 @@ common =
 
       VERSION_SUFFIX = suffix;
 
-      outputs = [ "out" "dev" "man" "doc" ];
+      outputs =
+        [ "out" "dev" ]
+        ++ lib.optionals enableDocumentation [ "man" "doc" ];
```

Comment:
```suggestion
      outputs = [ "out" "dev" ]
        ++ lib.optionals enableDocumentation [ "man" "doc" ];
```

Reply:
This would be inconsistent with the style of the rest of this file.